src/index.html

```diff
@@ -32,7 +33,8 @@ <h1>Excepteur Sint Occaecat</h1>
         </a>
       </li>
     </ul>
-    <h2 class="point">Duis Aute</h2>
+    <h2 class="point">Pictures</h2>
```

Comment:
1) Hi, the heading should be immediately before the `<img>`, not the paragraph.

2) You should place the icon to the left-hand side of the text Pictures, like this: 📷Pictures

You can just move the `<i>` tag inside the `<h2>`.